src/containers/tutor-home-page/general-info-step/SelectGroup.tsx

```diff
@@ -1,30 +1,89 @@
-import React from 'react'
-import { FormControl, InputLabel, Select, MenuItem } from '@mui/material'
+import React, { useEffect, useState } from 'react'
+import {
+  FormControl,
+  InputLabel,
+  Select,
+  MenuItem,
+  SelectChangeEvent
+} from '@mui/material'
 import { useTranslation } from 'react-i18next'
 import { useSelectGroupStyles } from './SelectGroup.styles'
+import { LocationService } from '~/services/location-service'
 
 const SelectGroup: React.FC = () => {
   const { t } = useTranslation()
   const classes = useSelectGroupStyles()
+  const [countries, setCountries] = useState<string[]>([])
+  const [cities, setCities] = useState<string[]>([])
+  const [selectedCountry, setSelectedCountry] = useState('')
+  const [selectedCity, setSelectedCity] = useState('')
+
+  useEffect(() => {
+    const fetchCountries = async () => {
+      try {
+        const response = await LocationService.getCountries()
+        setCountries(response.data as string[])
```

Comment:
add `import { AxiosResponse } from 'axios'`
```suggestion
        const response: AxiosResponse<string[]> = await LocationService.getCountries()
        setCountries(response.data)
```

Reply:
![image](https://github.com/Space2Study-UA-1156/Client-01/assets/93076661/94f9cb4f-05d7-433d-8e9c-dbfd37b61dde)


It breaks down the application

It works if I use it like here

![image](https://github.com/Space2Study-UA-1156/Client-01/assets/93076661/5ef28825-6196-44f6-b907-d679d2be6414)

Reply:
It's strange. For me it works without casting.
![image](https://github.com/Space2Study-UA-1156/Client-01/assets/67462207/eadc6cd4-5f68-4d80-a01a-6d0ab5118a6a)
![image](https://github.com/Space2Study-UA-1156/Client-01/assets/67462207/0321d75c-2bdb-40c1-8dbf-8831a3e53915)

Reply:
What about adding a Response type for the service instead of adding it for each component that consumes that service?  

![image](https://github.com/Space2Study-UA-1156/Client-01/assets/56509285/23572f3d-0222-4097-84c7-9c768bd59a47)

![image](https://github.com/Space2Study-UA-1156/Client-01/assets/56509285/c47b8cbb-3bf5-4b82-a9f4-ab0c9488ac1f)